src/qml/pages/node/NodeSettings.qml

```diff
@@ -13,73 +13,72 @@ Item {
     id: nodeSettings
     property alias navMiddleDetail: nodeSettingsView.navMiddleDetail
     property alias navRightDetail: nodeSettingsView.navRightDetail
+
     StackView {
         id: nodeSettingsView
         property alias navMiddleDetail: node_settings.navMiddleDetail
         property alias navRightDetail: node_settings.navRightDetail
+        anchors.fill: parent
+
         initialItem: Page {
             id: node_settings
             property alias navMiddleDetail: navbar.middleDetail
             property alias navRightDetail: navbar.rightDetail
             background: null
+            implicitWidth: 490
             header: NavigationBar {
                 id: navbar
             }
             ColumnLayout {
-                spacing: 0
-                width: parent.width
-                ColumnLayout {
-                    spacing: 20
-                    Layout.maximumWidth: 450
-                    Layout.topMargin: 30
-                    Layout.leftMargin: 20
-                    Layout.rightMargin: 20
-                    Layout.alignment: Qt.AlignCenter
-                    Setting {
-                        Layout.fillWidth: true
-                        header: qsTr("Dark Mode")
-                        actionItem: OptionSwitch {
-                            checked: Theme.dark
-                            onToggled: Theme.toggleDark()
-                        }
+                spacing: 20
+                width: Math.min(parent.width, 490)
+                anchors.leftMargin: 20
+                anchors.rightMargin: 20
+                anchors.topMargin: 30
```

Comment:
the top margin for the columnlayout here doesn't seem to draw the same, PR on left, master on right:
![different-margins](https://user-images.githubusercontent.com/23396902/207969301-5fe437d6-c8cd-4377-aca1-2d785e4741bc.png)


Also in general, it appears that the left and right margins are different throughout the views now, but will confirm on macos.

Reply:
And the PR NodeSettings isn't max 450 here

Reply:
Great catch. I now see Page has its own padding property and is ignoring my anchor values. https://doc.qt.io/qt-6/qml-qtquick-controls2-control.html#padding-prop.

Reply:
those padding values apply to just the "contentItem"